Phonebook.Frontend/src/app/pages/dashboard/dashboard.component.ts

```diff
@@ -24,10 +31,19 @@ export class DashboardComponent implements OnInit {
   @Select(BookmarksState)
   public bookmarkedPersons$: Observable<Person[]>;
   public removedLastPersons: Person[] | null = null;
-  constructor(private store: Store, private cd: ChangeDetectorRef) {}
+  public drawerOpen: boolean = !this.breakpointObserver.isMatched('(max-width: 768px)');
+  public drawerMode: MatDrawerMode = 'side';
+  public smallerScreen: boolean = false;
+  constructor(private store: Store, private cd: ChangeDetectorRef, private breakpointObserver: BreakpointObserver) {}
 
   public ngOnInit() {
     this.changeOrder();
+    this.breakpointObserver
+      .observe('(max-width: 768px)')
+      .pipe(untilComponentDestroyed(this))
+      .subscribe(result => {
+        this.drawerMode = !result.matches ? 'side' : 'push';
+      });
```

Comment:
Wäre das mit der Verwendung der [breakpoints](https://github.com/T-Systems-MMS/phonebook/blob/master/Phonebook.Frontend/src/styles/media-queries.scss) nicht sinnvoller @T-Systems-MMS/phonebook-developers ?

Dann könnte sich der ganze Rest dazu auch gespart werden.

Reply:
geh in diesem fall nicht, weil wir keine variablen zwischen scss und typescript teilen können. Habe aber den gleiche wert genommen.

Reply:
ich nehme das mal als Ja :)